packages/client/components/navigation/Sidebar.tsx

```diff
@@ -152,24 +146,66 @@ export default function Dashboard() {
                 _focus={{ textDecoration: 'none', border: 'none' }}
                 transition="0.2s"
               >
-                <Text size="md">
-                  <FontAwesomeIcon icon={faClipboardList} />{' '}
-                  {t('components.navigation.dashboard.jobs_list')}
-                </Text>
+                <Flex alignItems="center">
+                  <ClipboardList />
+                  <Text ml="3" size="md">
+                    {t('components.navigation.dashboard.jobs_list')}
+                  </Text>
+                </Flex>
               </Link>
             </NextLink>
           </Stack>
         ) : null}
 
-        <Stack alignSelf="flex-end"></Stack>
+        {userPurpose === '/hire' && (
+          <Box p="4" textAlign="center">
+            <NextLink href={'/post-job'} passHref>
+              <Link>
+                <Button
+                  color="white"
+                  colorScheme="neutral"
+                  variant="morePadding"
```

Comment:
Hey @zhenyangg ! We're trying to standardize the use of the button variants across the site instead of adding so many props. Can you use `neutral` variant instead? But great job anyways, this actually looks cool

Reply:
As discussed, let's leave this as it is for now, since there's no way to combine multiple variants.